app/views/projects/bots/personal_access_tokens/_generate_personal_access_token_modal.html.erb

```diff
@@ -0,0 +1,135 @@
+                left-0
+                flex
+                items-center
+                pl-3
+                pointer-events-none
+              "
+            >
+              <%= viral_icon(
+                name: :calendar_days,
+                classes: "w-5 h-5 text-gray-500 dark:text-gray-400"
+              ) %>
+            </div>
+            <%= form.text_field :expires_at,
+                            placeholder: I18n.t("date.formats.iso"),
+                            autocomplete: "off",
+                            "data-datepicker-target": "datePicker",
+                            "data-datepicker-dialog": "true" %>
+          </div>
+        </div>
+
+        <div class="form-field">
+          <%= form.label :scopes %>
+          <% Irida::Auth.all_available_scopes.each do |scope| %>
+            <div class="flex">
+              <div class="flex items-center h-5">
+                <%= check_box_tag "personal_access_token[scopes][]",
+                scope,
+                id: "personal_access_token_scopes_#{scope}" %>
+              </div>
+              <div class="ml-2 text-sm">
+                <%= label_tag "personal_access_token_scopes_#{scope}",
+                scope,
+                class: "font-medium text-slate-900 dark:text-slate-300" %>
+                <p
+                  id="<%= "personal_access_token_scopes_#{scope}_help-text" %>"
+                  class="
+                    text-xs
+                    font-normal
+                    text-slate-500
+                    dark:text-slate-300
+                  "
+                ><%= t("auth.scopes.#{scope}") %></p>
+              </div>
+            </div>
+          <% end %>
+        </div>
+
+      </div>
+      <div class="mt-4">
+        <%= form.submit t(
+                      "projects.bots.index.bot_listing.generate_personal_access_token_modal.submit"
+                    ),
+                    class:
+                      "
+                                                                      focus:outline-none
+                                                                      text-sm
+                                                                      text-white
+                                                                      bg-primary-700
+                                                                      hover:bg-primary-800
+                                                                      focus:ring-0
+                                                                      rounded-md
+                                                                      p-2
+                                                                      dark:text-white
+                                                                      dark:bg-primary-600
+                                                                      dark:hover:bg-primary-700
+                                                                      cursor-pointer
```

Comment:
I noticed the button styling through the PR are different than most of what we have in the website (in dark mode), in case we wanted consistency by using `button button--size-default button--state-primary` and `button button--size-default button--state-default` for submit/cancel buttons.
I noticed this for the new bot account, new bot submit/cancel, and generate personal token submit/cancel buttons

Reply:
Thanks for reviewing. The reason the classes are different is because we are moving away from using those classes as its not the correct way to use ` @apply` in the tailwindcss file. I guess the accepted way is to use the css directly as seen on the flowbite page. @joshsadam can probably give you a way better explanation but that's my understanding of it